### DISCUSSION TITLE
Sources for Components?

### DISCUSSION THREAD

**Discussion by corymosiman12** [BuildingComponentLibrary/bcl-manifest#18] [Q&A]
Hey @kflemin - hope you're doing well!

I've recently been looking into the components hosted here, specifically around the Opaque materials and Construction Assemblies. The majority of the opaque components include this description: `Material from a data set created from ASHRAE HOF 2005 and ASHRAE 90.1-2004 Appendix A`

Do you have any documentation as to where / how these calculations were made? Any info you can provide would be appreciated - thanks!

**Comment by kflemin** [comment]
@corymosiman12! We miss you!  We'll tell you the answer if you come back 😄 jk.

This is from @nllong:
The HOF is the list of materials and their thermal properties, 90.1 Appendix A is the overall u-factor for assemblies. We took these data for each of the 90.1 construction types, determined the construction layers using properties from HOF, then removed the film coefficients (E+ calculates those itself), then calculated the value of the insulation that was needed to meet the overall assembly U-Factor. These were run in energyplus and the results were confirmed to match for each assembly.
 
HOF:
![HOF](https://user-images.githubusercontent.com/2205659/162069419-0bf84c85-375e-4ef2-9a6b-537454de9ab6.jpg)
 
90.1
 
![90 1](https://user-images.githubusercontent.com/2205659/162069457-6b6cda7d-b7aa-4205-aba4-0f2058c44e6d.jpg)

**Reply by corymosiman12** [reply]
@kflemin @nllong please tell me where I'm being dumb:
![image](https://user-images.githubusercontent.com/28991233/162086789-3a3c549f-d32b-406a-a7e9-10e5dc6f0c70.png)

these numbers no equal...

also, just to make sure I understand correctly, E+ just uses the simple parallel path method from HOF, correct? no isothermal planes method or modified zone method for heat transfer?

**Reply by nllong** [reply]
Take off the vertical wall film coefficients from the value in ashrae, then does it match?

**Reply by corymosiman12** [reply]
U = 0.292 btu/hr-ft2-F
R = 1/0.292 = 3.42
ri = 0.68
1/(3.42 - 0.68) = 0.36 ... ?

**Reply by nllong** [reply]
correct, the U would increase when removing film coefficients. I see that you are looking at just a single material, right? What about the entire construction object with all the layers?

**Reply by corymosiman12** [reply]
I guess this is what I'm trying to figure out. Many of the components from here are just single layer `Material` object types. We want to use them to create multi-layered constructions, but I'm just trying to go through the process right now of validating a few numbers / understand how you guys arrived at the conductivity values. 

I went back through my calcs, I had made a mistake in my math, but didn't solve the problem. I did a calc from scratch for a simple wall assembly:
- Indoor air film
- 5/8" drywall
- 3-1/2" Wood Stud @16"o.c. + air
- 5/8" drywall

Used HOF 2017 values and performed it using both the parallel path method and the isothermal planes method. My value comes close to yours, but still not sure how it relates back to the 90.1 values.

![image](https://user-images.githubusercontent.com/28991233/162217858-a36e46b9-d496-4a08-be5f-9258eb1a6d74.png)

IDF from [here](https://github.com/BuildingComponentLibrary/nrel-components/tree/develop/lib/components/Material/Opaque/Framing%20with%20Cavity/Wood-Framed%20-%204%20in.%20Studs%20-%2016%20in.%20OC%20-%20R-0%20Cavity%20Insulation/files)

Sry for being pesky...I know you a busy guy
